### ISSUE TITLE
Tagged template string considered an expression, not a function call

### ISSUE THREAD

**Issue by yoshuawuyts** [feross/standard#822]
### Intro
Template strings are a way to interpolate variables with strings. It's nice, and often more readable than traditional string concatenation

Tagged template strings take this behavior a bit further, and allow functions to be called as if they were strings - but instead the functions receive multiple strings, and a trailing array of positional arguments that the have been used for interpolation. This allows operating on the string and adding all sorts of interesting behavior.

### Example
```js
var css = require('sheetify')

// this is a static expression
`
  .js-yellow { color: #f7df1e }
  .bg-js-yellow { background-color: #f7df1e }
`

// this is a function call, which in this case has side effects
css`
  .js-yellow { color: #f7df1e }
  .bg-js-yellow { background-color: #f7df1e }
`
```

### Bug
It appears that `standard@9` has introduced a regression where tagged template strings are now considered static expressions - even though they're functions. I think we should probably look into this.

```txt
❯ standard
standard: Use JavaScript Standard Style (http://standardjs.com)
  /Users/anon/src/yoshuawuyts/berlinjs/index.js:14:1: Expected an assignm
ent or function call and instead saw an expression.
```
Thanks!

**Comment by feross** [comment]
Thanks for the issue, @yoshuawuyts. This sounds like a bug in ESLint's [`no-unused-expressions`](http://eslint.org/docs/rules/no-unused-expressions) rule. I'll open an issue with them.

**Comment by yoshuawuyts** [comment]
Thank you :D

**Comment by feross** [comment]
Issue filed here: https://github.com/eslint/eslint/issues/8268

**Comment by feross** [comment]
They actually already have a PR out that adds an option to ignore tagged template literals: https://github.com/eslint/eslint/pull/8253 When that makes it into a release, I'll add the option and publish a new version on the v9 line.

**Comment by vitorbal** [comment]
@feross FYI, today's release will most likely contain this fix!

**Comment by feross** [comment]
@vitorbal Excellent! Just saw that. I'll do a `standard` as soon as the ESLint release is out. 👍

**Comment by feross** [comment]
Fix published as 9.0.2.

(Also, for users of the v10 beta, published as 10.0.0-beta.2.)

**Comment by romaindurand** [comment]
Hi, I still get an error on this line : 

    errors.push[`Missing Parameter : ${requiredParams.name}`]

I'm using VSCode and this extension : https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs
that uses my global install of standard

From the doc :

    To use JavaScript Standard Style in this workspace please install standard using 'npm install standard' or globally using 'npm install -g standard'.
    You need to reopen the workspace after installing standard.)

the error I'm getting is :

    [standard] Expected an assignment or function call and instead saw an expression. (no-unused-expressions)
    let errors: any[]
